solace-spring-cloud-stream-binder/solace-spring-cloud-stream-binder-core/src/main/java/com/solace/spring/cloud/stream/binder/provisioning/SolaceQueueProvisioner.java

```diff
@@ -185,6 +185,12 @@ private Queue provisionQueue(String name, boolean isDurable, EndpointProperties
 		return queue;
 	}
 
+	private void validateQueueName(String name) {
+		if (!StringUtils.hasText(name)) {
+			throw new ProvisioningException(String.format("The name of the queue to provision is invalid. At least one character is required. Current value is '%s'", name));
+		}
+	}
```

Comment:
Validation should be moved into `SolaceProvisioningUtil.getQueueNames()`. The SpEL expressions should always resolve a non-empty value regardless if provisioning is enabled or not.